### PR TITLE
feat: created command update from domains

### PR DESCRIPTION
**Pull request by maxwelbm**
- command update from domains

### CODE REVIEW

File: messages/update/domains/errors.go [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package domains
+
+import "errors"
+
+var (
+	ErrorUpdateDomain           = errors.New("Failed to update the Domain: %s. Check your settings and try again. If the error persists, contact Azion support.")
+	ErrorActiveFlag             = errors.New("Invalid --active flag provided. The flag must have  'true' or 'false' values. Run the command 'azion domains <subcommand> --help' to display more information and try again.")
+	ErrorDigitalCertificateFlag = errors.New("Invalid --digital-certificate-id flag provided. The flag must have an Integer or 'null' as a value. Run the command 'azion domains <subcommand> --help' to display more information and try again")
+	ErrorConvertDomainID        = errors.New("The domain ID you provided is invalid. The value must be an integer. You may run the 'azion list edge-application' command to check your application ID")
```
Comment (PatrickMenoti) [inline messages/update/domains/errors.go:9]: `azion list domains`
`to check your domain ID`

File: messages/update/domains/messages.go [hunk 2]
```diff
@@ -0,0 +1,18 @@
+package domains
+
+var (
+	Usage                    = "domains --domain-id <domain_id> [flags]"
+	ShortDescription         = "Updates a Domain"
+	LongDescription          = "Updates a Domain based on its ID to update its name and other attributes"
+	FlagDomainID             = "Unique identifier of the Domain"
+	FlagDigitalCertificateID = "Unique identifier of the Digital Certificate; this value is either an integer or null"
+	FlagApplicationID        = "Unique identifier for an edge application used by this domain.. The '--application-id' flag is mandatory"
```
Comment (PatrickMenoti) [inline messages/update/domains/messages.go:9]: `FlagDomainID`
`The '--domain-id' flag is mandatory`

File: messages/update/domains/messages.go [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+	FlagName                 = "The Domain's name"
+	FlagCnames               = "Cnames of your Domain"
+	FlagCnameAccessOnly      = "Whether the Domain should be Accessed through Cname only or not"
+	FlagIn                   = "Given path and JSON file to automatically update the Domain attributes; you can use - for reading from stdin"
+	OutputSuccess            = "Updated Domain with ID %d\n"
+	FlagActive               = "Whether the Domain should be active or not"
+	HelpFlag                 = "Displays more information about the update subcommand"
```
Comment (PatrickMenoti) [inline messages/update/domains/messages.go:16]: Displays more information about the update domains subcommand

File: messages/update/domains/messages.go [hunk 4]
```diff
@@ -4,9 +4,9 @@ var (
 	Usage                    = "domains --domain-id <domain_id> [flags]"
 	ShortDescription         = "Updates a Domain"
 	LongDescription          = "Updates a Domain based on its ID to update its name and other attributes"
-	FlagDomainID             = "The '--domain-id' flag is mandatory"
+	FlagDomainID             = "The '--domain-id'"
 	FlagDigitalCertificateID = "Unique identifier of the Digital Certificate; this value is either an integer or null"
-	FlagApplicationID        = "Unique identifier for an edge application used by this domain.. The '--application-id' flag is mandatory"
+	FlagApplicationID        = "Unique identifier for an edge application used by this domain. The '--application-id'"
```
Comment (PatrickMenoti) [inline messages/update/domains/messages.go:9]: The '--application-id'
you can remove this as well

**Review by PatrickMenoti — APPROVED** [review]

**Review by gabriel-azion — APPROVED** [review]

**Pull request merged**